common/src/command.rs

```diff
@@ -94,22 +106,15 @@ impl<T: CommandType> CommandTypeDynamic for T {
 }
 
 #[derive(Debug, Clone)]
-pub struct Speak(pub String);
-
-#[derive(Debug, Clone)]
-pub struct SpeechPriority(pub Priority);
+pub struct Speak(pub String, pub Priority);
 
 impl CommandType for Speak {
```

Comment:
It's so we can use command handlers like I described. When we have many variants of the command enum, it will become unwieldy to handle in one function.

Reply:
Then you are making a semantic difference about _where_ I put the handlers. Considering resolved for now.